### PR TITLE
boilplate setup with styles and redux and router and firebase

### PR DESCRIPTION
**Pull request by JimmyMcBride**
# Description
Set's up file structure for project, redux, firebase hosting, and react router

File Structure:
**src |**
  **actions |** _// place for putting action files for our redux_
  **components |**
    **Auth |** _// where auth related components go_
      **Login |**
        -index.js _// naming main files to be exported index.js for cleaner imports (ex: import Login from "./components/Auth/Login" you do not need to add '/index' on the end because it's implied when you name a file index or something like that)_
      **Register |** _// same as Login, but for the register page_
        -index.js
    -index.js _// index file for Auth folder_
    **Global |** _// folder for global components and styles_
      -index.js _// global styled components (Button, Form, Input, etc) are imported from here_
      -theme.js _// main theme file to pull colors from, just some basic bootstrap colors are there for now_
    **HomePage |**
      -index.js
  **reducers |**
    -index.js _// combine all reducer files here_
    -initialState.js _// we store our initial state for the app here and import into each reducer file we create_
    -loginReducer.js _// an example of a reducer file that will import initial state and then export this file to combine reducers in index.js that gets imported into index file for the whole app and passes down to all children through using connect and mapStateToProps_
  -App.js _// creating our routes for everything in here, this contains the whole app and all routes, contains routes for home, login and register_
  -index.css _// this file sets html font size to 2vw for ultimate responsiveness_
  -index.js _// sets up redux, react router, calls our normailze.css and bootstrap css files for project and also contains a currently unregistered service worker_

Fixes # (issue)
No noticeable issues yet

## Type of change
added boilerplate for project

Please delete options that are not relevant.

- [x] This change requires a documentation update _// added all dependencies to read me_

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Ran server and tested live hosting site as well as locally that all components function properly 

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings _// only warning for empty and unused action and reducer but that will soon not be an issue_
- [x] I have added tests that prove my fix is effective or that my feature works _// tested server live and locally_
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```
Comment (dave-vazquez) [inline public/manifest.json:25]: PWA ready! :tada:

File: src/components/Auth/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import styled from "styled-components";
+
+import { Button, Form, Input, ButtonWrapper } from "../../Global";
+import { Linkton } from "../../Global";
```
Comment (dave-vazquez) [inline src/components/Auth/Login/index.js:6]: just curious what the 'n' in Linkton is

File: src/components/Global/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import theme from "./theme";
+import { Link } from "react-router-dom";
+import { Button as Btn, Form as Frm, Input as Inp } from "reactstrap";
```
Comment (dave-vazquez) [inline src/components/Global/index.js:4]: 🎉

**Review by dave-vazquez — APPROVED** [review]
Looks dope. We should add Randy's linter at some point.

**Review by chazr88 — APPROVED** [review]
Seems fine.

**Review by aalejandromr — APPROVED** [review]

**Pull request merged**